
human_link_lengths: [0.569,0.194,0.328,0.285,0.357,0.285,0.45]
human_link_radii: [0.12,0.05,0.1,0.04,0.03,0.04,0.03]
